app/services/versioned_files_service/versions_manifest.rb

```diff
@@ -77,24 +92,23 @@ def version_metadata
     end
 
     def versions
-      versions_hash.keys
+      manifest[:versions].keys
     end
 
-    private
-
-    attr_reader :path
-
     def manifest
       @manifest ||= (path.exist? ? JSON.parse(@path.read).with_indifferent_access : {}).tap do |manifest|
+        manifest[:@version] ||= 1
```

Comment:
Calling this version is likely to cause confusion.

Reply:
Would you be happier with `"$schema": "....#v1"`? That struck me as super over-kill, but maybe it's less confusing